Movies-MVP/Movies-MVP/Modules/MovieDetail/Cells/CastCell/CastTableViewCell.swift

```diff
@@ -11,13 +11,18 @@ final class CastTableViewCell: UITableViewCell, UICollectionViewDataSource {
         static let actorCellID = "ActorCollectionCell"
     }
 
+    private enum Constants {
+        static let errorMessage = "init(coder:) has not been implemented"
+    }
+
     // MARK: - Public Properties
 
     var movieId = Int()
 
     // MARK: - Private properties
 
-    private var cast: [CastResult.Cast] = []
+    private var cast: [Cast] = []
```

Comment:
```suggestion
    private var cast: [Cast] = []
```
var должен быть под let